src/test/ui/numeric/uppercase-base-prefix-invalid-no-fix.rs

```diff
@@ -0,0 +1,33 @@
+#![allow(unused_variables)]
```

Comment:
If a variable name begins with `_` the compiler won't complain if it's not used. So you can use `let _a = ...` and remove this `allow` attribute.

Even better, you can just use `let _ = ...`.

Even better, you can just use `_ = ...`. (This one is a fairly recent language addition.)

Reply:
Okay! This was just to make the file look like `uppercase-base-prefix.rs`, but I agree that `_ = ...` looks better. Will fix.